src/main/java/globalquake/ui/stationselect/StationSelectFrame.java

```diff
@@ -28,8 +30,12 @@ public StationSelectFrame(DatabaseMonitorFrame databaseMonitorFrame) {
         JButton toggleButton = new JButton("<");
         selectAll = new JButton(new SelectAllAction(databaseMonitorFrame.getManager()));
         deselectAll = new JButton(new DeselectAllAction(databaseMonitorFrame.getManager()));
+        searchBar = new SearchBar();
         selectAll.addActionListener(this);
         deselectAll.addActionListener(this);
+        suggestionPanel.setVisible(false);
+        suggestionPanel.setBorder(BorderFactory.createLineBorder(Color.BLACK, 1));
+        suggestionPanel.setPreferredSize(new Dimension(375, 50));
```

Comment:
The sizing of the `suggestionalPanel` has to be changed to dynamically adjust for changes in the number of suggestions and/ or a successful search.

Reply:
I would try to make the suggestion panel constant size and find a way that it is displayed on top of the existing UI, instead of moving the existing components

Reply:
Hey, I was wondering if you could expand on this and maybe provide an example or an image of what the Suggest Panel should look like. Do you think I should make it its own JFrame?